app/DynamicGraphs/GraphNodeUtils.hs

```diff
@@ -11,56 +11,56 @@ import Database.Requirement (Modifier (..), Req (..))
 -- | Converts the given number of credits and list of modifiers into a string
 -- | in readable English
 -- | Assumes each modifier constructor appears in modifiers at most once
--- | The MODOR constructor may appear more than once, but each occurrence
--- | of MODOR contains exactly one constructor for all its elements
--- | and such constructor does not appear anywhere else in MODAND
+-- | The ModOr constructor may appear more than once, but each occurrence
+-- | of ModOr contains exactly one constructor for all its elements
+-- | and such constructor does not appear anywhere else in ModAnd
 stringifyModand :: Float -> [Modifier] -> String
 stringifyModand creds modifiers = let
-    dept = maybeHead [x | DEPARTMENT x <- modifiers]
+    dept = maybeHead [x | Department x <- modifiers]
     deptFormatted = case dept of
         Nothing -> ""
         Just x -> ' ' : x
-    depts = maybeHead [xs | MODOR xs@((DEPARTMENT _):_) <- modifiers]
+    depts = maybeHead [xs | ModOr xs@((Department _):_) <- modifiers]
     deptsFormatted = case depts of
         Nothing -> ""
         Just xs -> ' ' : concatModor xs
-    level = maybeHead [x | LEVEL x <- modifiers]
+    level = maybeHead [x | Level x <- modifiers]
     levelFormatted = case level of
         Nothing -> ""
         Just x -> " at the " ++ x ++ " level"
-    levels = maybeHead [xs | MODOR xs@((LEVEL _):_) <- modifiers]
+    levels = maybeHead [xs | ModOr xs@((Level _):_) <- modifiers]
     levelsFormatted = case levels of
         Nothing -> ""
         Just xs -> " at the " ++ concatModor xs ++ " level"
-    raw = maybeHead [x | REQUIREMENT (RAW x) <- modifiers]
+    raw = maybeHead [x | Requirement (Raw x) <- modifiers]
     rawFormatted = case raw of
         Nothing -> ""
         Just x -> " from " ++ x
-    raws = maybeHead [xs | MODOR xs@((REQUIREMENT (RAW _)):_) <- modifiers]
+    raws = maybeHead [xs | ModOr xs@((Requirement (Raw _)):_) <- modifiers]
     rawsFormatted = case raws of
         Nothing -> ""
         Just xs -> " from " ++ concatModor xs
 
     in show creds ++ deptFormatted ++ deptsFormatted ++ " FCEs"
         ++ levelFormatted ++ levelsFormatted ++ rawFormatted ++ rawsFormatted
 
--- | Formats a MODOR into FCEs string
--- | Assumes all modifiers in the MODOR have the same constructor
+-- | Formats a ModOr into FCEs string
+-- | Assumes all modifiers in the ModOr have the same constructor
 formatModor :: Float -> [Modifier] -> String
```

Comment:
Right, so this function should be renamed `formatModOr` (and similar with the `concatModor` function below)